### DISCUSSION TITLE
User Profile Read API (v7.0.0) and No Storage of Personally Identifiable Information (PII) – Any Options to Exclude Details?

### DISCUSSION THREAD

**Discussion by ankit1shahu** [Sunbird-Lern/Community#173] [Query] [answered]
we utilize the User Profile Read API to retrieve user profiles for observation, surveys, and projects. Starting from version 7.0.0, we have made the decision not to store users' personally identifiable information (PII) during profile reading. Are there any methods, such as using flags or keys, to exclude specific details when making API calls? The API URL is v5/user/read/$UserId.

**Comment by Jayaprakash8887** [comment, marked as answer]
@ankitpws No, there is no such read API which controls the returned fields in the API.
